src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
 
 export default class Fall_GM extends React.Component {
 	render() {
-		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT 
-	
-		const renderHumanReadableTime = ({ days, hours, minutes, seconds, completed }) => {
-			if (completed) return <h1 id="time-to-gm-headline-now">Fall General Meeting 2020 is happening right now!</h1>; //EVAN TODO: add the rest of the program and links to zoom / slack here
-	
-			let dayString ="Day";
-			let hourString = "Hour";
-			let minuteString = "Minute";
-			let secondString = "Second";
-	
-			if (days !== 1) dayString += 's';
-			if (hours !== 1) hourString += 's';
-			if (minutes !== 1) minuteString += 's';
-			if (seconds !== 1) secondString += 's';
+		const FALL_GM_2020_START_TIME = new Date('2020-10-05T18:30:00-0700'); //October 5th 2020 at 6:30 PDT
+
+		const renderCountdownOrNow = ({ days, hours, minutes, seconds, completed }) => {
+			if (completed) return <GM_NOW/>;
+			else {
+				let dayString = "Day";
+				let hourString = "Hour";
+				let minuteString = "Minute";
+				let secondString = "Second";
+				if(days !== 1) dayString += "s";
+				if(hours !== 1) hourString += "s";
+				if(minutes !== 1) minuteString += "s";
+				if(seconds !== 1) secondString += "s";
+
+				return <GM_COUNTDOWN days={days} hours={hours} minutes={minutes} seconds={seconds}
+				dayString={dayString} hourString={hourString} minuteString={minuteString} secondString={secondString}/>
+			}
+		};
 	
+		return (
+			<div id="fall-gm-page">
+				<Banner decorative />
+				<Countdown date={FALL_GM_2020_START_TIME} renderer={renderCountdownOrNow} />
+			</div>
+		);
+	}
+}
+
+class GM_NOW extends React.Component{
+	renderCommitteeSpeakers() {
```

Comment:
This strategy is slightly DRYer than having a bunch of repeated HTML elements, it also makes changes easier if a committee decides to swap speakers

Reply:
I wonder if there's a bit of de-dup we can do with the constants in `src/config`, but no worries if that's too much of a headache?